README.md

```diff
@@ -67,24 +67,12 @@ flowchart LR
 
 ## Getting Started
 
-The [Getting started with KinD][tut-kind] tutorial explains how to get started
+The [Getting Started](docs/getting-started.md) tutorial explains how to get started
 with Dracon.
 You can also access our community contributed pipelines
-[here](https://github.com/ocurity/dracon-community-pipelines)
+[here](https://github.com/ocurity/dracon-community-pipelines).
```

Comment:
maybe for a ticket: shall we deprecate community pipelines?
the og idea was that community pipelines are a searchable registry where we upload complete use-cases for dracon. this has not picked up interest so maybe we can merge them back into examples

Reply:
Will raise a 🎫

Reply:
@northdpole raised OCU-188